package.json

```diff
@@ -113,5 +114,20 @@
       "client_src/*"
     ],
     "delay": "2500"
+  },
+    "node_deb": {
```

Comment:
Shall we expose this into a file?

Reply:
It didn't seem to be an option with node-deb but I'll check as I'd much rather.

Reply:
Ok nvm I think you're right. No worries!